QuizApp/Presenter/Coordinators/AppRouter.swift

```diff
@@ -43,16 +43,20 @@ class AppRouter: AppRouterProtocol{
         let qvc = QuizzesViewController(router: self)
         let navQvc = UINavigationController(rootViewController: qvc)
         let svc = SettingsViewController(router: self)
-        
-        //tabbar[QuizzesViewControler, SettingsViewController]
+        let searchVc = SearchViewController(router: self)
+        let navSearchVc = UINavigationController(rootViewController: searchVc)
```

Comment:
You do not need to wrap each VC into UINavigationController for this type of application design.